pkg/cmd/cli/version/version.go

```diff
@@ -1,81 +0,0 @@
-/*
```

Comment:
@scaranoj this file should not be deleted. Thanks @ashish-amarnath for catching that.